app/controllers/clients/coordinated_entry_assessments_controller.rb

```diff
@@ -40,6 +40,10 @@ def new
 
     def destroy
       @assessment.destroy
+      if @assessment.active?
+        previous_completed = @client.ce_assessments.completed.order(submitted_at: :desc).first
+        previous_completed.update(active: true) if previous_completed.present?
```

Comment:
Let's move this on to  the class `@assessment.ensure_active`, then wrap this in a transaction with the destroy.  It looks odd to me to call this after destroy, but I think you have to.